pycmds/_record.py

```diff
@@ -1065,27 +712,27 @@ def create_main_tab(self):
         # display settings
         input_table = pw.InputTable()
         input_table.add("Display", None)
-        allowed_values = [device.name for device in self.control.devices]
-        self.device_combo = pc.Combo(allowed_values=allowed_values)
-        self.device_combo.updated.connect(self.on_update_device)
-        input_table.add("Device", self.device_combo)
+        allowed_values = [sensor.name for sensor in self.control.sensors]
+        self.sensor_combo = pc.Combo(allowed_values=allowed_values)
+        self.sensor_combo.updated.connect(self.on_update_sensor)
+        input_table.add("sensor", self.sensor_combo)
```

Comment:
```suggestion
        input_table.add("Sensor", self.sensor_combo)
```

Most of our labels are capitalized